tests/test_environment_BondOrder.py

```diff
@@ -96,6 +96,35 @@ def test_repr(self):
         bo = freud.environment.BondOrder(1.5, 0, 12, 6, 6)
         self.assertEqual(str(bo), str(eval(repr(bo))))
 
+    def test_ref_points_ne_points(self):
+        lattice_size = 10
+        # big box to ignore periodicity
+        box = freud.box.Box.square(lattice_size*5)
+        angle = np.pi/30
+        ref_points, points = util.make_alternating_lattice(lattice_size, angle)
+
+        # actually not used
```

Comment:
Is this indicating that these values aren't actually used by BondOrder? Because you are still passing them in.

Reply:
They are not used by `BondOrder` internally, whatever values are given for the constructor.